include/dicom/dicom.h

```diff
@@ -411,6 +389,100 @@ void dcm_log_debug(const char *format, ...);
 DCM_EXTERN
 const char *dcm_get_version(void);
 
+/**
+ * An enum of Value Representations.
+ *
+ * Value Representations which are not known to libdicom will be coded as
+ * DCM_VR_ERROR (unknown Value Representation).
+ *
+ * Note to maintainers: this enum must match the table in dicom-dict.c, and
+ * the DcmVRTag enum. As the DICOM standard evolves, numbering must be
+ * maintained for ABI compatibility.
+ */
+typedef enum _DcmVR {
```

Comment:
There's a formatted copy of the docs here:

http://www.rollthepotato.net/~john/libdicom/

Reply:
Thanks for the link!

It looks like these private type definitions get included into the API documentation: http://www.rollthepotato.net/~john/libdicom/api.html#c._DcmVR

It would be nice if we could configure Sphinx or Hawkmoth to ignore declarations with a leading underscore. My understanding is that they are considered private and shouldn't be relied up by users.

Reply:
Regarding the empty API doc page, is there an issue with the API doc compilation?

Reply:
I've not updated that copy of the formatted docs for a little while, I'll update it now. You can generate the docs on your machine too, of course.

Reply:
OK, updated now.

Reply:
I've updated the formatted docs again for these renames.